lib/sites.js

```diff
@@ -0,0 +1,50 @@
+export const allSites = [
+  {
+    key: 1,
+    pos: [53.577587, 9.991238],
```

Comment:
the common name for such data would be `coordinates`. 
if you want to keep it, i recommand writing the whole name for better readability.

Reply:
I will change to coordinates!